cmd/veneur-emit/main.go

```diff
@@ -87,7 +93,22 @@ func main() {
 	}
 	logrus.Debugf("destination: %s", addr)
 
-	if *mode == "metric" {
+	if *shellCommand {
+		var conn MinimalClient
+		conn, err = statsd.New(addr)
+		if err != nil {
+			logrus.WithError(err).Fatal("Error!")
+		}
+
+		command := strings.Join(flag.Args(), " ")
```

Comment:
As we discussed, let's cut out the middle-sh and execute the command directly (: